### DISCUSSION TITLE
Assembly

### DISCUSSION THREAD

**Discussion by rob-devoted** [iamromulan/cellular-modem-wiki#11] [General - Quectel]
Did the assembly section of the documentation get removed?

**Comment by MickMorley** [comment]
Yes, you can see it when you select Activity.  I see the device used was updated as well.  @iamromulan commented that "Random PCI link interruptions seen to occur..." with the MR5210 from Hangzhou WildChip Tech Co.

From this one
![image](https://github.com/iamromulan/quectel-rgmii-configuration-notes/assets/19698105/2d13f2b6-385c-41d7-953a-19660863ea55)

To this one
![image](https://github.com/iamromulan/quectel-rgmii-configuration-notes/assets/19698105/c1b43fee-4575-4612-8c6c-1f3c6cdd7f0a)

**Reply by rob-devoted** [reply]
Thanks Mick!

**Comment by iamromulan** [comment]
Yes I apologize. I am currently on vacation in Italy but I will get back to this again.

The MCU boards are manufactured inconsistently and I have seen many issues with the PCIe link between the modem and the ethernet chipset of the board becoming interrupted. It happens in certain RF and/or Electromagnetic scenarios. It's pretty much a sure thing to happen quickly when you are on the move with it. 

I cannot recommend it. The original single sim board's stability is what I will have a link to. I cannot recommend the MCU board any longer.

**Reply by rob-devoted** [reply]
Thanks for the info. I just received the MCU MR5210 on Friday and have been testing it. I'm definitely seeing some random connectivity issues, but unsure if they're specific to the PCIe link. I assumed it was switching towers or something. I will also order the new board you recommended so I can have both.

You mentioned "when you are on the move with it" -- are you able to use these while in a car? I wasn't sure about their ability to automatically/gracefully switch between towers.

Thanks again

**Reply by iamromulan** [reply]
> Thanks for the info. I just received the MCU MR5210 on Friday and have been testing it. I'm definitely seeing some random connectivity issues, but unsure if they're specific to the PCIe link. I assumed it was switching towers or something. I will also order the new board you recommended so I can have both.
> 
> You mentioned "when you are on the move with it" -- are you able to use these while in a car? I wasn't sure about their ability to automatically/gracefully switch between towers.
> 
> Thanks again

It should absolutely gracefully switch between towers or reconnect upon getting a signal back 

When it happens; don't reboot; instead plug in by USB and start an adb shell session.

Run the command dmesg


In the output it'll be pretty obvious.
The time stamp it initially occurs:
msm_pcie_handle_linkdown: PCIe link is down for RC0

Then repeating until reboot: 
eth0: cmd = 0xff, should be 0x47
.
eth0: pci link is down

**Reply by MickMorley** [reply]
Thank you for keeping us in the loop with your testing.  I have several MR5210 and just ordered my first "5G M.2 To RJ45 Kit from YILEXUN last night.
